src/components/FormAlertWrapper.js

```diff
@@ -0,0 +1,115 @@
+import _ from 'underscore';
+import {View} from 'react-native';
+import PropTypes from 'prop-types';
+import React from 'react';
+import {withNetwork} from './OnyxProvider';
+import Icon from './Icon';
+import * as Expensicons from './Icon/Expensicons';
+import RenderHTML from './RenderHTML';
+import TextLink from './TextLink';
+import Text from './Text';
+import colors from '../styles/colors';
+import compose from '../libs/compose';
+import OfflineIndicator from './OfflineIndicator';
+import networkPropTypes from './networkPropTypes';
+import styles from '../styles/styles';
+import withLocalize, {withLocalizePropTypes} from './withLocalize';
+
+const propTypes = {
+    /** Wrapped child components */
+    children: PropTypes.node.isRequired,
+
+    /** Styles for container element */
+    containerStyles: PropTypes.arrayOf(PropTypes.object),
+
+    /** Whether to show the alert text */
+    isAlertVisible: PropTypes.bool.isRequired,
+
+    /** Whether message is in html format */
+    isMessageHtml: PropTypes.bool,
```

Comment:
When would there be a message in HTML?

Reply:
It looks like it's only used in one spot `this.props.reimbursementAccount.errorModalMessage` and I really don't like it. I can live with it, but I don't like it. I don't like that there is a component with a prop that is only used for a single use case. For me, that means we should have a separate component like `<FormHTMLAlertWrapper>`.

I would also be extatic if we could make a backend change so that there is no HTML message coming from the server at all. I don't think the server should be responding with HTML.

Reply:
I'm pretty determined to remove this, but it may take some changes in Secure first. Lets address this after this PR

Reply:
OK, awesome. Thanks!